src/components/panel/panel.scss

```diff
@@ -0,0 +1,20 @@
+.md-panel-container {
+  align-items: center;
+  display: flex;
```

Comment:
The `.md-panel-container` going to be the thing that holds the arbitrary `template`, right? If so, I wouldn't make it `display:flex`; they layout of the panel's content should be determined by the template being loaded.